doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-debugging-tips.rst

```diff
@@ -0,0 +1,68 @@
+.. _tests-integration-testing-teuthology-debugging-tips:
+
+Analysing and Debugging A Teuthology Job
+-----------------------------------------
+
+For scheduling an integration test please refer to, `Scheduling Test Run`_
+Here, we will be discussing how to analyse failed/dead jobs to root cause the problem and amend it.
+
+Triaging the cause of failure
+------------------------------
+
+Once a teuthology run is successfully completed, we can access the results using
+pulpito dashboard for example:
+
+which might look something like:
+
+http://pulpito.front.sepia.ceph.com/<job-name>/<job-id>/
+
+To triage, open the teuthology log for it using either:
+
+http://pulpito.front.sepia.ceph.com/<job-name>/<job-id>/teuthology.log
+
+or via sshing into teuthology server using::
+
+    ssh teuthology.front.sepia.ceph.com
+
+and then opening log file with signature as:
+
+   /a/<job-name>/<job-id>/teuthology.log
+
+for example in our case::
+
+  nano /a/ideepika-2020-11-03_04:03:28-rados-wip-yuri-testing-2020-10-28-0947-octopus-distro-basic-smithi/5585704/teuthology.log
+
+Generally, a job failure is recorded in teuthology log as a Traceback which gets
+added to job summary.  While analysing a job failure, we generally start looking
+for ``Traceback`` keyword and further see the call stack and logs that might had
+lead to failure Most of the time, traceback will also be including the failing
```

Comment:
missing a period before "Most":
lead to failure Most of the time, traceback will also be including the failing